src/Terminus.php

```diff
@@ -294,7 +295,6 @@ private function addBuiltInCommandsAndHooks()
     {
         // List of all hooks and commands. Update via 'composer update-class-lists'
         $this->commands = [
-            'Consolidation\\Filter\\Hooks\\FilterHooks',
```

Comment:
Is there any reason why this got removed?

Reply:
updated via the "composer add command"